integrations/opennms-vmware/src/main/resources/META-INF/opennms/component-dao.xml

```diff
@@ -33,7 +33,6 @@
     </bean>
 
     <bean id="vmwareConfigDao" class="org.opennms.netmgt.dao.vmware.jaxb.VmwareConfigDaoJaxb">
-        <property name="configResource" ref="vmwareConfigResourceLocation"/>
```

Comment:
How about remove the vmwareConfigResourceLocation ?

Reply:
That is not used , I can remove.

Reply:
It seems still there
`
    <bean id="vmwareConfigResourceLocation" class="java.lang.String">
        <constructor-arg value="file:${opennms.home}/etc/vmware-config.xml"/>
    </bean>
`